holoviews/core/util.py

```diff
@@ -346,6 +346,8 @@ def tree_attribute(identifier):
     These custom attributes start with a capitalized character when
     applicable (not applicable to underscore or certain unicode characters)
     """
+    if not identifier:
```

Comment:
This would match anything that is falsey even though it should only ever match the empty string. I would prefer to be explicit, i.e.:

`if identifier=='':`

Reply:
I can update this when I add a unit test or two

Reply:
That's fine with me, though the rest of the function will then error if someone were to pass 0 or None; not sure if that's any better behavior, but maybe?

Reply:
A 0 or None should be error conditions imho: those shouldn't be passed as identifiers in any situation.

Reply:
Right; I just mean that this isn't like a Parameterized object that's actually checking for such things; 0 and None will just error later at some point. Anyway, I'm fine with however you want to handle that!